### ISSUE TITLE
Plugged in USB devices are not recognized on battery

### ISSUE THREAD

**Issue by sudoforge** [linrunner/TLP#575]
[x] I've read and accepted the [Bug Reporting Howto](https://github.com/linrunner/TLP/blob/master/.github/Bug_Reporting_Howto.md)
[x] I've attached all required `tlp-stat` outputs via [Gist](https://gist.github.com/) (see below)

**Describe the bug**

**Expected behavior**

**To Reproduce**

Steps to reproduce the unexpected behavior:

**1. Does the problem occur on battery or AC or both?**

It only occurs on battery.

**2. Actions to reproduce the behaviour**

Create a file at `/etc/tlp.d/01-usb.conf` with the following contents:

```
USB_AUTOSUSPEND=0
```

Comment out any lines containing `USB_AUTOSPEND` from `/etc/tlp.conf`. Then:

* Disconnect from AC power.
* Insert a USB device.
* Note that it is not recognized via `lsusb` or any programs that should, e.g. `udiskie`.
* Run `sudo systemctl restart tlp.service`
* Note that the device is now recognized.
* Remove the device, and then re-insert it.
* Note that the device is no longer recognized.

**3. Shell commands entered and their output**
N/A

**4. Full output of `tlp-stat` via https://gist.github.com/ for *all* matching cases of 1.**

https://gist.github.com/sudoforge/dd9cdbbaf2a0ac95b7c722a016725d4c

**Additional context**

To summarize the issue:

```
➜ for f in /etc/tlp.d/01-usb.conf /etc/tlp.conf; do printf "from ${f}:\n\t"; rg 'USB_AUTOSUSPEND\b' "$f"; done
from /etc/tlp.d/01-usb.conf:
        6:USB_AUTOSUSPEND=0
from /etc/tlp.conf:
        323:#USB_AUTOSUSPEND=1


➜ sudo systemctl restart tlp.service
```

When on battery mode, _with a USB device already plugged in_, restarting the service sends power to the USB device. However, after removing the USB device and re-inserting it, it does not receive power.

**Comment by sudoforge** [comment]
> Where's the full output then?

Based on the template, it was assumed that the only relevant information that would be necessary is the configuration and usb related output from `tlp-stat`; this is why the gist referenced in the original comment contains the command line and full output from `tlp-stat -c -u`. Do you need the entire output of the command, sans any options? I suppose `-e` would be useful too.

> try to add the USB3 Controllers (check the output of tlp-stat -e for by USB controller, xhci_hcd ) to RUNTIME_PM_BLACKLIST

I'll give this a go later today.

> Although it is mentioned again and again in some forums, this is not the right command for the purpose. Use
> 
> ```
> tlp start
> ```

```
➜ systemctl cat tlp.service
# /usr/lib/systemd/system/tlp.service
# tlp - systemd startup/shutdown service
#
# Copyright (c) 2020 Thomas Koch <linrunner at gmx.net> and others.
# This software is licensed under the GPL v2 or later.

[Unit]
Description=TLP system startup/shutdown
After=multi-user.target NetworkManager.service
Before=shutdown.target
Documentation=https://linrunner.de/tlp

[Service]
Type=oneshot
RemainAfterExit=yes
ExecStart=/usr/bin/tlp init start
ExecStop=/usr/bin/tlp init stop

[Install]
WantedBy=multi-user.target
```
